src/Eloquent/Traits/HasAdjacencyList.php

```diff
@@ -12,6 +12,21 @@
 use Staudenmeir\LaravelAdjacencyList\Eloquent\Relations\RootAncestorOrSelf;
 use Staudenmeir\LaravelAdjacencyList\Eloquent\Relations\Siblings;
 
+/**
```

Comment:
@staudenmeir this (similar for `src/Eloquent/Traits/HasGraphAdjacencyList.php`) does have impact on user-land. I think the docblocks are correct and they could be useful, but I would understand if you don't want to ship this change lightly, or don't want to include this change. In that case you could remove these and move them to the relevant test classes (User and Node)